qnm_filter/bilby_helper.py

```diff
@@ -82,6 +82,8 @@ def set_bilby_ifo(
     sampling_frequency,
     duration,
     start_time,
+    minimum_frequency,
```

Comment:
is it worth having default values?

Reply:
I'd rather not. The desired value may vary from case to case. The current way can remind ourselves each time it's used.